pkg/trace/stats/quantile/converter.go

```diff
@@ -0,0 +1,171 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package quantile
+
+import (
+	"errors"
+	"sort"
+
+	"github.com/DataDog/sketches-go/ddsketch/mapping"
+	"github.com/DataDog/sketches-go/ddsketch/pb/sketchpb"
+	"github.com/golang/latestprotobuf/proto"
+)
+
+// ddSketch represents the sketch described here: http://www.vldb.org/pvldb/vol12/p2195-masson.pdf
+// This representation only supports positive values.
+type ddSketch struct {
+	// bins is the map from index --> count
+	bins map[int32]float64
+	// contiguousBins is a more compact representation for contiguous bins.
+	// the index of each bin is the index in the array + contiguousBinsOffset
+	contiguousBins       []float64
+	contiguousBinsOffset int
+	// zeros is the count of 0 and its close neighbors (close defined by gamma).
+	zeros int
+	// mapping is the mapping of the ddSketch: interpolation, gamma and global index offset.
+	mapping mapping.IndexMapping
+}
+
+// count returns the count for a given index.
+func (s *ddSketch) count(index int) (count int) {
+	if index >= s.contiguousBinsOffset && index < s.contiguousBinsOffset+len(s.contiguousBins) {
+		count = int(s.contiguousBins[index-s.contiguousBinsOffset])
+	}
+	if c, ok := s.bins[int32(index)]; ok {
+		count += int(c)
+	}
+	return count
+}
+
+func (s *ddSketch) maxSize() int {
+	return len(s.bins) + len(s.contiguousBins)
+}
+
+// getIndexes returns all the sorted indexes contained in s1 and s2.
+func getIndexes(s1 ddSketch, s2 ddSketch) []int {
+	// todo: No need to re-allocate that array at each conversion.
```

Comment:
```suggestion
	// TODO(x): No need to re-allocate that array at each conversion.
```
Nit: Usually we use `TODO(github_username)` or `TODO(x)` if anyone else can pick it up.